pkg/hubble/observer/local_observer.go

```diff
@@ -248,6 +248,20 @@ func (s *LocalObserverServer) GetFlows(
 		}).Debug("GetFlows finished")
 	}()
 
+	// if the node name does not match the node filter then we will never return
+	// any flows so return immediately to close the stream
+	nodeName := nodeTypes.GetName()
+	nodeNameFilter, err := filters.NewNodeNameFilter(req.Whitelist, req.Blacklist)
+	if err != nil {
+		return err
+	}
+	if !nodeNameFilter.Match(nodeName) {
+		log.WithFields(logrus.Fields{
+			"node_name": nodeName,
```

Comment:
whitelist and blacklist might be useful info for debugging.

Reply:
Good idea. However, the blacklist and whitelist will already be printed in the log message printed by the `defer` statement in the previous block of code.